lib/util.js

```diff
@@ -488,7 +488,7 @@ function formatPrimitiveNoColor(ctx, value) {
 
 
 function formatError(value) {
-  return '[' + Error.prototype.toString.call(value) + ']';
+  return value.stack || '[' + Error.prototype.toString.call(value) + ']';
```

Comment:
Added fallback in case the error is some custom error that isn't capturing the stack correctly.